src/Avalonia.Base/Input/GestureRecognizers/ScrollGestureRecognizer.cs

```diff
@@ -66,8 +66,7 @@ public void Initialize(IInputElement target, IGestureRecognizerActionsDispatcher
         
         public void PointerPressed(PointerPressedEventArgs e)
         {
-            if (e.Pointer.IsPrimary && 
-                (e.Pointer.Type == PointerType.Touch || e.Pointer.Type == PointerType.Pen))
+            if (e.Pointer.Type == PointerType.Touch || e.Pointer.Type == PointerType.Pen)
```

Comment:
Are you sure that we want to intercept non-primary pointers for scrolling?

Reply:
We want, but after discussion it's better to revert it from this PR. I will come back to it in the future some day.